tensorflow/core/kernels/matmul_op_impl.h

```diff
@@ -586,6 +588,16 @@ struct LaunchBatchMatMul<GPUDevice, Scalar> {
                                     std::is_same_v<Scalar, Eigen::bfloat16>;
     using Coefficient = std::conditional_t<is_16bit_input, float, Scalar>;
 
+    se::blas::CallContext call_context =
+      se::blas::CallContext::kNone;
+    OP_REQUIRES(context, grad_x == false || grad_y == false,
+                    errors::InvalidArgument("Both grad_x and grad_y can't be set to True"));
```

Comment:
Internal review comment:

Please change the error message to "At least 1 of grad_x and grad_y shall be false"

Reply:
Done!